packages/hierarchy-builder/src/hierarchy-builder/HierarchyDefinition.ts

```diff
@@ -110,6 +110,8 @@ export interface DefineHierarchyLevelProps {
   parentNode: HierarchyDefinitionParentNode | undefined;
   /** Optional hierarchy level filter. */
   instanceFilter?: GenericInstanceFilter;
+  /** Optional hierarchy level size limit. Default limit is 1000. */
+  hierarchyLevelSizeLimit?: number;
```

Comment:
Why do we need this attribute on `DefineHierarchyLevelProps`?

Reply:
`DefineHierarchyLevelProps` is the type of props in `setupObservables`. Then it is passed to `createPreProcessedNodesObservable` where `hierarchyLevelSizeLimit` attribute is passed to `applyLimit` and `_queryReader.read` functions. I just assumed that I can change `DefineHierarchyLevelProps`.
Changed so it does not have this attribute. Now `setupObservables` and `createPreProcessedNodesObservable` have props of type: `DefineHierarchyLevelProps & { hierarchyLevelSizeLimit?: number | "unbounded" }`.